### DISCUSSION TITLE
Add site to Panopticon, Web Services - Installer

### DISCUSSION THREAD

**Discussion by komir2004** [akeeba/panopticon#263] [Q&A (Community Support)]
Hi, I have a problem with adding Joomla 4 on Panopticon.
When I tried to save it, I got an error message saying "The “Web Services - Installer” plugin is disabled on your site".
![Akeeba-Panopticon-2](https://github.com/akeeba/panopticon/assets/17553018/ce1affde-0658-40ff-a8d5-41649e57a04d)

However, when I checked my site settings, I saw that the plugin was enabled.
I try to disable and enable again plugin but no luck

I have successfully added several sites to the Panopticon without any issues. The process was smooth and straightforward, and I did not encounter any errors or bugs.
## Environment information (please fill in this information)

* Panopticon version [e.g. 1.2.3]: Akeeba Panopticon 1.0.3
* PHP version [e.g. 8.2.0]: 8.1.24
* Database type [e.g. MySQL, MariaDB]: MySQL
* Database version: 8.0.35
* Browser [e.g. Chrome, Safari]: Edge
* Browser version: 119

**Comment by brianteeman** [comment]
Please follow ALL the advice in the message!!

**Comment by komir2004** [comment]
I appreciate your help with this issue. I  forgot to mention earlier that I have already attempted to debug the problem, but I did not receive any further information.
The page in question is an upgrade from Joomla 3 to Joomla 4, which was done without any errors. I am not sure if this is relevant or not, but I thought I would share it with you. The page is running smoothly on Joomla 4.

**Comment by komir2004** [comment]
Hi any news regarding this problem?

**Reply by nikosdion** [reply]
Please do follow the troubleshooting instructions https://github.com/akeeba/panopticon/wiki/Managing-Sites#troubleshooting-the-connection This is the total sum of problems you can possibly run into.

**Comment by komir2004** [comment]
Solved.
Added "api" inside "Allow direct access, including .php files, to these directories inside" in ".htaccess Maker" (Admin Tools Professional)

**Reply by nikosdion** [reply]
This is **DEFINITELY** not necessary. The `api` folder is explicitly allowed in the code generated by the .htaccess Maker which the user cannot change since Joomla! 4 was still in beta back in 2020.
